08_deploy_bmo.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/bash
 
+#source common.sh
 eval "$(go env)"
 
 # Get the latest bits for baremetal-operator
 export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
 
-source common.sh
-
-oc new-project bmo-project
+# Make a local copy of the baremetal-operator code to make changes
+cp -r $BMOPATH/deploy ocp/.
+sed -i 's/bmo-project/openshift-machine-api/g' ocp/deploy/role_binding.yaml
```

Comment:
This will pattern will need to change when https://github.com/metalkube/baremetal-operator/pull/88 is merged. We can do that separately if this PR merges first.